source/service/Epidemic.ts

```diff
@@ -74,16 +74,30 @@ export interface Province extends Base, StatisticData {
     cities?: City[];
 }
 
+export type Area = 'city' | 'continent' | 'country' | 'province';
+export type AreaData = Base &
+    Partial<Record<`${'city' | 'province'}_${StatisticType}Count`, number>> &
+    Record<`${Area}EnglishName`, string> &
+    Record<`${Area}Name`, string> &
+    Record<`${'city' | 'province'}_zipCode`, string>;
 export async function getOverall() {
     const { body } = await epidemic.get<Overall[]>('Overall', { Range: '0-9' });
 
     return body;
 }
 
 export async function getHistory() {
-    const { body } = await epidemic.get<Province[]>('Area', { Range: '0-9' });
+    const { body } = await epidemic.get<AreaData[]>('Area', { Range: '0-199' });
 
-    return body;
+    const updatedBody = body.map(item => ({
+        ...item,
+        provinceShortName: item.provinceName,
+        confirmedCount: item.province_confirmedCount,
+        suspectedCount: item.province_suspectedCount,
+        curedCount: item.province_curedCount,
+        deadCount: item.province_deadCount
+    }));
+    return updatedBody as unknown as Province[];
```

Comment:
```suggestion
    const updatedBody = body.map(item => ({
        provinceShortName: item.provinceName,
        confirmedCount: item.province_confirmedCount,
        suspectedCount: item.province_suspectedCount,
        curedCount: item.province_curedCount,
        deadCount: item.province_deadCount
    }));
    return updatedBody as Province[];
```
`map()` 在数学上是一个集合到另一个集合的**映射**，目标数据结构上没有的字段，硬加上能不类型报错吗？

Reply:
已修正，加入Province类型包含的字段
 id: item.id,
 updateTime: item.updateTime,